test/composables/useSidebar.itemsByPaths.test.ts

```
@@ -0,0 +1,891 @@
+import { describe, expect, it } from 'vitest'
+import type { DefaultTheme } from 'vitepress'
+import { useSidebar } from '../../src/composables/useSidebar'
+import realSpec from '../../docs/public/openapi.json'
+
+function cleanResult(result: (DefaultTheme.SidebarItem | DefaultTheme.SidebarItem[])): any {
+  if (Array.isArray(result)) {
+    return result.map(cleanResult)
+  }
+
+  if (result.text && result.text.includes('<span')) {
+    delete result.text
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)